File: core/src/main/java/com/ustadmobile/core/view/SELQuestionEditView.java
package com.ustadmobile.core.view;


/**
 * View represents editing SEL Questions that will be asked as per the SEL tasks.
 * SELQuestionEdit Core View extends Core UstadView. Will be implemented
 * on various implementations.
 */
public interface SELQuestionEditView extends UstadView {

    //View name
    String VIEW_NAME = "SELQuestionEdit";

    /**
     * Closes the view.
     */
    void finish();


}

File: core/src/commonMain/kotlin/com/ustadmobile/staging/core/view/SELQuestionEditView.kt
package com.ustadmobile.staging.core.view

import com.ustadmobile.core.view.UstadView


/**
 * View represents editing SEL Questions that will be asked as per the SEL tasks.
 * SELQuestionEdit Core View extends Core UstadView. Will be implemented
 * on various implementations.
 */
interface SELQuestionEditView : UstadView {

    /**
     * Closes the view.
     */
    fun finish()

    companion object {

        //View name
        val VIEW_NAME = "SELQuestionEdit"
    }


}
